packages/flutter_web_plugins/lib/src/navigation/utils.dart

```diff
@@ -2,11 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
-
 import 'dart:html';
 
-AnchorElement _urlParsingNode;
+AnchorElement? _urlParsingNode;
```

Comment:
Statics are lazy-initialized by the language. We don't need to hand-code lazy initialization. We can do:

```
final AnchorElement _urlParsingNode = AnchorElement();
```

Reply:
#TIL this is lazily-initialized!